docs/src/main/asciidoc/security-keycloak-admin-client.adoc

```diff
@@ -146,6 +147,8 @@ public class RolesResource {
 }
 ----
 
+For more details consult https://www.keycloak.org/documentation[Keycloak API documentation].
```

Comment:
```suggestion
For more details consult https://www.keycloak.org/docs/latest/server_development/#admin-rest-api[Keycloak Admin REST API documentation].
```

Reply:
This link was on the main sec page but we lost it during the refactoring/introducing a new guide

Reply:
@sberyozkin updated